code/modules/modular_computers/file_system/programs/generic/email_client.dm

```diff
@@ -1,7 +1,7 @@
 /datum/computer_file/program/email_client
-	filename = "emailc"
-	filedesc = "Email Client"
-	extended_desc = "This program may be used to log in into your email account."
+	filename = "messengerc"
```

Comment:
if you're changing the filename, you'll also need to change the filename in code\modules\modular_computers\computers\subtypes\preset_pda.dm. Line 23, needs to be updated.

Reply:
Got it thanks, applied and pushed those 2 changes.